### DISCUSSION TITLE
navigation bar not rendering correctly on mobile webpage

### DISCUSSION THREAD

**Discussion by cchanzl** [mmistakes/minimal-mistakes#3146] [General]
Steps to replicate issue:
1. Open https://ifoadatascienceresearch.github.io/about/ on a mobile device
2. See that there is a large blank space between "Site Navigation" to "About"

Issue disappears when i remove the bottom code from my .md file.

sidebar:
  title: "Site Navigation"
  nav: about-nav

any help on this would be appreciated.

Thanks!

**Comment by iBug** [comment]
It's your own fault: <https://github.com/IFoADataScienceResearch/IFoADataScienceResearch.github.io/blob/e0d095874c3ac163ab00d8c593f6c15c44e2ef1d/_includes/sidebar.html#L2>

Use some reasonable responsive CSS instead of the ugly hack to make it behave properly on mobile devices.

**Comment by mmistakes** [comment]
Remove this CSS that's added as inline `style`. That's what is making the huge white space. The sidebar is still there, it's just not visible, so adding height to it pushes everything that comes after it down the page.

```
style="height: 10000px;"
```

![image](https://user-images.githubusercontent.com/1376749/132858731-5824fd4f-a3a8-48f6-ad31-653bcbe0a6e2.png)

**Comment by cchanzl** [comment]
brilliant. thank you both for your prompt response! it is resolved now. great template :)
